test/user.c

```diff
@@ -62,7 +62,7 @@ test_kevent_user_get_hires(struct test_context *ctx)
     kev.fflags &= ~NOTE_FFCTRLMASK;
     kev.fflags &= ~NOTE_TRIGGER;
     kev.flags = EV_CLEAR;
-    kevent_get_hires(&ret, ctx->kqfd);
+    kevent_get_hires(&ret, ctx->kqfd, 5);
```

Comment:
This increases the timeout massively, from 0.0005 seconds to 5 seconds?  Why not just pass a timespec in?